tensorboard/components/tf_imports/BUILD

```diff
@@ -31,6 +32,19 @@ tf_web_library(
     ],
 )
 
+tensorboard_webcomponent_library(
+    name = "polymer_lib",
+    srcs = [":polymer"],
+    visibility = ["//visibility:public"],
+    destdir = "tf-imports",
+    deps = [
+        ":polymer_externs_lib",
+        "//third_party/javascript/polymer/v1/web-animations-js:other-js",
```

Comment:
Assuming intentional that this is v1 while others are v2.

Reply:
There isn't v2 version of web-animations-js :(